CaPPMS/Model/FaqInformation.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
```

Comment:
You shouldn't need the system and system.collections.generic directives and you aren't using any annotations so I don't think you need that one either.

Reply:
It says i need it for the created of the GUID